src/System.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -18,13 +18,17 @@ static DataTestUtility()
             NpConnStr = Environment.GetEnvironmentVariable("TEST_NP_CONN_STR");
             TcpConnStr = Environment.GetEnvironmentVariable("TEST_TCP_CONN_STR");
 
-            if (string.IsNullOrEmpty(NpConnStr) || string.IsNullOrEmpty(TcpConnStr))
+            if (!AreConnStringsSetup())
             {
-                Console.WriteLine("WARNING: Test connection strings not defined! Tests cannot be run. ***");
-                Environment.Exit(0);
+                Console.WriteLine("WARNING: Test connection strings not defined! Tests cannot be run.");
```

Comment:
Can we please avoid outputting a warning in this case as it causes noise in our default builds as it gets picked up by msbuild as a warning.